utils/create_dependency_mapping.py

```diff
@@ -7,10 +7,15 @@ def topological_sort(dependencies):
     new_dependencies = {}
     graph = defaultdict(list)
     for node, deps in dependencies.items():
+        node_name = node.split("/")[-2]
         for dep in deps:
+            dep_name = dep.split(".")[-2]
+            if dep_name == node_name:
+                # Skip self dependencies for topological sort as they create cycles
+                continue
```

Comment:
This is the case in ijepa for example, where we import `IJepaConfig` in modular

Reply:
that's quite an edge case!